roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -1,50 +1,41 @@
 ---
+- name: Load lib_openshift modules
+  include_role:
+    name: lib_openshift
+
 - fail:
     msg: "The openshift_manageiq role requires OpenShift Enterprise 3.1 or Origin 1.1."
   when: not openshift.common.version_gte_3_1_or_1_1 | bool
 
-- name: Copy Configuration to temporary conf
-  command: >
-    cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{manage_iq_tmp_conf}}
-  changed_when: false
-
 - name: Add Management Infrastructure project
   command: >
     {{ openshift.common.client_binary }} adm new-project
     management-infra
     --description="Management Infrastructure"
-    --config={{manage_iq_tmp_conf}}
+    --config={{ openshift_master_config_dir }}/admin.kubeconfig
```

Comment:
We are currently copying the kubeconfig to a tmp file.  This shouldn't affect the admin.kubeconfig.